man/man5/zfs-module-parameters.5

```diff
@@ -1637,6 +1637,26 @@ OpenZFS pre-release versions and now have compatibility issues.
 Default value: \fB0\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_forced_export_unmount\fR (int)
+.ad
+.RS 12n
+During forced unmount, leave the filesystem in a disabled mode of operation,
+in which all new I/Os fail, except for those required to unmount it.
+Intended for users trying to forcibly export a pool even when I/Os are in
+progress, without the need to find and stop them.  This option does not
+affect processes that are merely sitting on the filesystem, only those
```

Comment:
What does "sitting on" mean?  Is this referring to having open file handles or the CWD (current working directory)?  Does this mean that regardless of what `zfs_forced_export_unmount` is set to, we can unmount filesystems that have open fd's?  Does that require the `-f` flag to `zfs unmount`/`zfs destroy`?

Reply:
No.  For more detail see my reply to your top-level question.